### DISCUSSION TITLE
Bash tab uses WSL instead of Git Bash

### DISCUSSION THREAD

**Discussion by Mart-Bogdan** [gitextensions/gitextensions#11511] [Q&A]
### Environment

- GitExtensions version: 4.2.1.17611
- GIT version:  2.43.0.windows.1
- OS version:  Windows 10.0.19045.3803
- .NET version: 6.0.25
- **WSL:**  Ubuntu Ubuntu-22.04. Kernel 5.15.133.1-microsoft-standard-WSL2

### Issue description

![image](https://github.com/gitextensions/gitextensions/assets/775038/6759e4c5-3897-454e-9ac1-2eca7cf9558b)

If WSL is installed on system, it adds bash.exe into system path. It's located at: 

C:\Windows\System32\bash.exe and C:\Users\%USER%\AppData\Local\Microsoft\WindowsApps\bash.exe

So when Git Ext tries to launch git bash, it launches WSL one, not one that is bundled with git, which is C:\Program Files\Git\cmd\git-bash.exe

In settings I just chose bash.

I didn't noticed it at first, but when I started using project with GIT LFS, it wasn't configured under WSL and it caused warning in console when I run git commands, and checked out file link instead of actual files. 

Related to #7523: Allow custom shell in Console tab

### Steps to reproduce

1. Install git extensions
2. Install WSL 2 
wsl --set-default-version 2
wsl.exe --install Ubuntu-22.04

### Did this work in previous version of GitExtensions?

I don't know

**Comment by gerhardol** [comment]
OK for me:
![image](https://github.com/gitextensions/gitextensions/assets/6248932/06aa4afa-eaf6-470d-b788-32ddad91f9ed)

Have you set the path?
![image](https://github.com/gitextensions/gitextensions/assets/6248932/3c6680e3-6d43-4145-a3b9-9e3abeb3d103)

I believe bash.exe in Linux tools path if set is attempted first, then PATH

If path is \\wsl$, wsl Git is used by default
https://git-extensions-documentation.readthedocs.io/en/main/settings.html#wsl-git-notes

**Comment by Mart-Bogdan** [comment]
Thanks, @gerhardol it works.

This is git ext settings and won't alter git setings? Nice then.

Would be nice still to be able to do it automatially. And or add option to shell settings

![image](https://github.com/gitextensions/gitextensions/assets/775038/15b914e0-e135-4ccd-a232-cbca433f13fc)
